edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -1,428 +1,441 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why is this diff showing up so large? Did we change some formatting or something? I cannot tell which lines were changes... can you please revert?

Reply:
not sure.. nothing I did changed the formatting.. Going to go ahead and revert